src/components/Compiler/Compiler.ts

```diff
@@ -36,7 +41,7 @@ export class Compiler extends BaseCompiler {
         const classConverter = new ClassConverter();
         data = classConverter.getData(data);
 
-        const namespace = data.namespace.replace(".", "/");
+        const namespace = data.namespace.replace(/\./g, "/");
```

Comment:
Replacing globally so we get a directory like `com/example/namespace/` instead of `com/example.namespace`. Maybe we can make this configurable if theres a preference for the more flattened folder structure?

Reply:
I like the idea of making it configurable, I'll open issue for this.